database.sql

```diff
@@ -61,3 +61,144 @@ ALTER TABLE `item`
+
+
+-- SQL REQUESTS FOR COMIC_CULT DATABASE -- 
+
+------------------------------------------------------
+-- IF YOU HAVE TO DROP ALL THE DATABASE             --
+------------------------------------------------------
+
+DROP DATABASE comic_cult;
+
+------------------------------------------------------
+-- CREATION AND USE REQUEST                         --
+------------------------------------------------------
+
+CREATE DATABASE comic_cult;
+
+USE comic_cult;
+
+------------------------------------------------------
+-- REQUEST FOR 'role' TABLE CREATION                --
+------------------------------------------------------
+
+CREATE TABLE role (
+	id INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+    role VARCHAR(80) NOT NULL
+    );
+
+------------------------------------------------------
+-- REQUEST FOR 'author' TABLE CREATION              --
+------------------------------------------------------
+
+CREATE TABLE author (
+	id INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+	first_name VARCHAR(80) NOT NULL,
+    last_name VARCHAR(100) NOT NULL,
+    birth_date DATE,
+    editor VARCHAR(100),
+    biography TEXT
+    );
+
+-------------------------------------------------------
+-- REQUEST FOR 'role_author' JUNCTION TABLE CREATION --
+-------------------------------------------------------
+    
+CREATE TABLE role_author (
+	author_id INT NOT NULL,
+	CONSTRAINT fk_role_author
+		FOREIGN KEY (author_id)
+		REFERENCES author(id),
+	role_id INT NOT NULL,
+	CONSTRAINT fk_author_role
+		FOREIGN KEY (role_id)
+		REFERENCES role(id)
+	);
+
+------------------------------------------------------
+-- REQUEST FOR 'category' TABLE CREATION            --
+------------------------------------------------------
+
+CREATE TABLE category (
+	id INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+    category VARCHAR(80) NOT NULL
+    );
+
+------------------------------------------------------
+-- REQUEST FOR 'user' TABLE CREATION                --
+------------------------------------------------------
+
+CREATE TABLE `user` (
+	id INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+	user_name VARCHAR(80) NOT NULL,
+    first_name VARCHAR(80) NOT NULL,
+    last_name VARCHAR(80) NOT NULL,
+    email VARCHAR(100) NOT NULL,
+    password VARCHAR(80) NOT NULL,
+    birth_date DATE NOT NULL,
+    is_admin BOOL NOT NULL
+    );
+
+------------------------------------------------------
+-- REQUEST FOR 'comic_book' TABLE CREATION          --
+------------------------------------------------------
+
+CREATE TABLE comic_book (
+	id INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+    title VARCHAR(255) NOT NULL,
+    isbn BIGINT,
+    date_of_release DATE,
+    pitch TEXT NOT NULL,
+    nb_pages INT,
+    volume INT,
+    price FLOAT,
+    cover BLOB,
+    author_name VARCHAR(100),
+	category_id INT NOT NULL,
+	CONSTRAINT fk_comic_book_category
+		FOREIGN KEY (category_id)
+		REFERENCES category(id),
+	user_id INT,
+	CONSTRAINT fk_comic_book_user
+		FOREIGN KEY (user_id)
+		REFERENCES user(id)
```

Comment:
No don't add a constraint on the comic_book table with the user. At the start it will be annoying to deal with.